LayoutTests/http/wpt/2dcontext/imagebitmap/nullptrcrash-when-gpu-process-times-out.html

```diff
@@ -0,0 +1,24 @@
+<script>
+globalThis.testRunner?.waitUntilDone();
```

Comment:
Can't we use `testRunner.dumpAsText();` so the expected file can have only this text

```
Test passes if it does not crash.
```